### ISSUE TITLE
Skip link text fails accessibility contrast check

### ISSUE THREAD

**Issue by microbit-matt-hillsdon** [microsoft/pxt-microbit#6198]
I think this is a scenario where we should be aiming for something very clear.

![Image](https://github.com/user-attachments/assets/675eb02a-5c9e-4001-be11-0a04738c3f97)

**Comment by abchatra** [comment]
Theming related issue may be?

**Comment by srietkerk** [comment]
I can still look at this, from quick investigation, it doesn't look like a theming issue.
